.github/workflows/update-schema-dependency-version.yaml

```diff
@@ -0,0 +1,55 @@
+name: Update Schema Version
+
+on:
+  workflow_call:
+    inputs:
+      major:
+        description: 'Schema Major version'
+        type: string
+        required: true
+      minor:
+        description: 'Schema Minor version'
+        type: string
+        required: true
+      patch:
+        description: 'Schema Patch version'
+        type: string
+        required: true
+      downstream-repo-to-update:
+        description: 'The name of the downstream repository for which to update the schema version'
+        type: string
+        required: true
+      schema-property-name-in-repo:
+        description: 'The property name in the repos pom.xml for the schema version, e.g. rcsb.mojave.model.version'
+        type: string
+        required: true
+
+
+env:
+  NEW_SCHEMA_VERSION: "${{ inputs.major }}.${{ inputs.minor }}.${{ inputs.patch }}"
+
+jobs:
+  update-version:
+    runs-on:
+      - self-hosted
+      - java
+    steps:
+      - name: Checking out ${{ inputs.downstream-repo-to-update }}
+        uses: actions/checkout@v3
+        with:
+          repository: rcsb/${{ inputs.downstream-repo-to-update }}
+          path: ${{ inputs.downstream-repo-to-update }}
+      - name: Setup git author information
+        run: |
+          git config user.name RCSBSystem
+          git config user.email it@rcsb.org
+      - run: mvn --version
+      - name: Set new schema dependency version
+        run: |
+          mvn versions:set-property -Dproperty=${{ inputs.schema-property-name-in-repo }} -DnewVersion=$NEW_SCHEMA_VERSION -DgenerateBackupPoms=false
+      - name: Commit and tag
+        run: |
+          git add pom.xml
+          git commit -m "[actions] Updated schema dependency version to $NEW_SCHEMA_VERSION"
```

Comment:
Forgot to mention - there is a guard that prevents from infinitive loop, when maven automatically commits new version, [here ](https://github.com/rcsb/devops-cicd-github-actions/blob/4a5d72510db0d34e712fab3278588b8db3050cf0/.github/workflows/push-to-master.yaml#L15)

We should replace `[actions]` with something like `[automatic-schema-release]`. Think that will be clear then

Reply:
Thanks, now added